src/Components/CreateNewPoi/PoiFourthPageFields.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import '../ComponentsCSS/createPoiForm.scss';
+
+const mapStateToProps = state => ({
+  file: state.pois.file,
+});
+const poiFourthPageFields = ({
+  dispatch, file,
+}) => (
+  <div className="poi-create">
+    <div className="poi-name">
+      <input
+        type="file"
+        id="fileItem"
+        onChange={e => dispatch({ type: 'INSERT_PICTURE', file: URL.createObjectURL(e.target.files[0]) })}
```

Comment:
if dispatch doesn't work remove it

Reply:
PoiFourthPageFields.js removed because isn't already functionnal.